### ISSUE TITLE
Enable text compression ON, but reported OFF

### ISSUE THREAD

**Issue by dommm063** [GoogleChrome/lighthouse#9963]
#### Provide the steps to reproduce
1. Run LH on https://www.roadtrippin.fr
  

Report says that compression is not active on this site for php, css, js files.
```
Enable text compression
Text-based resources should be served with compression (gzip, deflate or brotli) to minimize total network bytes.
```
I checked with my provider and other tools, GZIP is enable (with htaccess file).

#### Environment Information
* Affected Channels: 
* Lighthouse version: 5.6.0
* Chrome version: 78.0.3904.97
* Node.js version: ?
* Operating System: Windows 10

**Related issues**

**Comment by Lofesa** [comment]
Hi @dommm063 
I have tested your url with LH Chrome same version you had, but LH version is 5.2, and can´t get these error.

![Captura](https://user-images.githubusercontent.com/22342793/68776165-edbc6d80-062f-11ea-8f88-0885c4d7c1d7.png)

![Captura](https://user-images.githubusercontent.com/22342793/68776453-60c5e400-0630-11ea-9c73-508de9546d7b.png)

**Comment by dommm063** [comment]
OK, but it seems to me that last version is best version, no ?

**Comment by patrickhulce** [comment]
I'm not able to reproduce this with 5.6.0 either. To @Lofesa's point, how are you running 5.6.0 in Chrome 78? That's not the Lighthouse version that ships with 78...

**Comment by Lofesa** [comment]
Hi @dommm063 
With LH 5.6 from web.dev , I get this:

![Captura](https://user-images.githubusercontent.com/22342793/68806617-781fc400-0666-11ea-8816-bc82c8c3e3ea.png)

**Comment by simgu** [comment]
I've seen the same issues with Lighthouse when testing our sites with Lighthouse CLI v5.6.0. Just to make sure our CDN is not doing something strange, I tested also using google.se and Lighthouse was reporting two static assets for the rule "Enable text compression". Loaded one of those files and checked the network tab in Chrome and the response had Content-Encoding: gzip.

How I executed LH (Node v12.13.0, Chrome 78.0.3904.108):
lighthouse https://google.se --view

**Comment by patrickhulce** [comment]
@simgu @Lofesa is it possible there's some intermediary proxy server on your network or machine that requests are run through where they could be mutated? I'm not able to reproduce these results on any of these URLs.

**Comment by Lofesa** [comment]
Hi @patrickhulce 
If you take a look at the images I have posted, I´m unable to reproduce the issue in chrome 78 or web.dev with the url in the 1st. post.
Tried   https://google.se as stated by @simgu and are unable to reproduce the issue in both chrome 78 and web.dev

**Comment by DanDiplo** [comment]
I'm running 80.0.3987.116 (Official Build) (64-bit) on Windows 10 and have experienced similar issues. I'm finding Lighthouse seems to be randomly reporting that Text Compression needs enabling on sites I've tested. By randomly I mean that I can run it a few times on the exact same page within a few minutes and it will _sometimes_ report it needs enabling and sometimes not - there's no discernible pattern. I can run it 5 times and it will report 3 times it's off and 2 times it is OK! All with the same URL.

I actually tested the https://www.roadtrippin.fr/ site mention in this report and found Lighthouse did report Text Compression was **off** the first time, but on the second time, then off the third time. See screenshot below showing it off:

![image](https://user-images.githubusercontent.com/6676168/75249842-f3992180-57ce-11ea-8d0e-b7c981435824.png)

On all the sites I've tested where it warns me that Text Compression is off then I know that GZip is enabled. I've confirmed this with multiple other tools and checked the headers. So I believe the issue is with Lighthouse, though I appreciate it's not easy to reproduce on demand. Intermittent issues are difficult to debug, I know. Is there any more information that would be useful to provide?

**Comment by patrickhulce** [comment]
Thanks for chiming in @DanDiplo! Is it possible there is some intermediate proxy/anti-virus/extension running that could be mutating results? I'm still 0/12 on getting that URL to report a failure for "Enable text compression". In the past when we've seen reports of this sort of thing happening it ended up being something that injects itself into Chrome and messes with results (the most unexpected one was a random HP Printer driver that for some reason decided it needed to hook into the Chrome network stack... 🤦‍♂)

**Comment by DanDiplo** [comment]
Hi @patrickhulce So I tested Chrome in incognito mode (with no extensions) and still had same intermittent results. However, I then tried disabling the virus checker they have installed on my work PC, which is [ESET Endpoint Security](https://www.eset.com/uk/business/endpoint-security-windows/download/) 7.1.2053 and this fixed the issue! I also noticed my page speed scores increased dramatically, too. I never realised how much it was interfering with network traffic, which isn't great! I will do more testing, but I definitely looks like this is the culprit. Thanks for the suggestion and for this very useful tool!
